client/device/cloudftd/create.go

```diff
@@ -154,7 +154,18 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 
 	// 8. wait for generate command available
 	var metadata Metadata
-	err = retry.Do(ctx, UntilGeneratedCommandAvailable(ctx, client, createOup.Uid, &metadata), *retry.NewOptionsWithLoggerAndRetries(client.Logger, 3))
+	err = retry.Do(
+		ctx,
+		UntilGeneratedCommandAvailable(ctx, client, createOup.Uid, &metadata),
+		retry.NewOptionsBuilder().
+			Message("waiting for Cloud FTD onboarding to be done").
```

Comment:
Waiting for FTD record to be created in CDO...

Reply:
done 👍